### DISCUSSION TITLE
Host Second Flutter Project on GitHub Pages

### DISCUSSION THREAD

**Discussion by MH0386** [community/community#63124] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi Everyone,

I have a personal website hosted on GitHub Pages with the URL https://mh0386.github.io/. I'm trying to host another repository called "graduation_project" at https://mh0386.github.io/graduation_project/, but all I'm getting is a white background. Can you help me troubleshoot this issue?

Thanks in advance

**Comment by queenofcorgis** [comment]
Hi there! 

Are you using Submodules for this?
